fdbcli/ChangeFeedCommand.actor.cpp

```diff
@@ -153,49 +156,47 @@ ACTOR Future<bool> changeFeedCommandActor(Database localDb,
 		printUsage(tokens[0]);
 		return false;
 	}
-	if (tokencmp(tokens[1], "list")) {
-		if (tokens.size() != 2) {
-			printUsage(tokens[0]);
-			return false;
-		}
-		wait(changeFeedList(localDb));
-		return true;
-	} else if (tokencmp(tokens[1], "register")) {
-		if (tokens.size() != 5) {
-			printUsage(tokens[0]);
-			return false;
-		}
 
-		KeyRange range;
-		if (tenantEntry.present()) {
-			range = KeyRangeRef(tokens[3], tokens[4]).withPrefix(tenantEntry.get().prefix);
-		} else {
-			range = KeyRangeRef(tokens[3], tokens[4]);
-		}
+	const CommandMap commandHandlers = {
```

Comment:
Not sure what happened, but I just commented and they are not showing up. So forgive me if this commit is commented twice.

Can we move this outside the `ACTOR`? So this is not being constructed repeatly when the function is called..